code/game/objects/structures/plasticflaps.dm

```diff
@@ -8,43 +8,39 @@
 	anchored = TRUE
 	layer = ABOVE_MOB_LAYER
 	CanAtmosPass = ATMOS_PASS_NO
-	var/state = PLASTIC_FLAPS_NORMAL
```

Comment:
If these defines are not used anymore, remove them please.

Reply:
That's odd because I removed it on my end. I'll check again.

Reply:
Is that on my end or are you asking someone else?

Reply:
I don't see define removal in the diff ? Talking about these : https://github.com/tgstation/tgstation/blob/cec26dcb45c4f05e4ccd57d339d3aeaa70c44d35/code/__DEFINES/construction.dm#L32

Reply:
I see. Thanks for showing me this file.

Reply:
@AnturK Should be good now